packages/common/src/services/audius-backend/AudiusBackend.ts

```diff
@@ -409,15 +409,18 @@ export const audiusBackend = ({
   }
 
   async function signData({ sdk, data }: { sdk: AudiusSdk; data: string }) {
-    const prefixedMessage = `\x19Ethereum Signed Message:\n${data.length}${data}`
-    const [sig, recid] = await sdk.services.audiusWalletClient.sign({
-      message: { raw: Buffer.from(prefixedMessage, 'utf-8') }
-    })
-    const r = Buffer.from(sig.slice(0, 32)).toString('hex')
-    const s = Buffer.from(sig.slice(32, 64)).toString('hex')
-    const v = (recid + 27).toString(16)
-    const signature = `0x${r}${s}${v}`
-    return { data, signature }
+    if (sdk.services.audiusWalletClient.account) {
```

Comment:
Also super nit but an early return of `{data: '', signature: ''}` is a little easier to read than putting the rest of the function in an if statement.